src/node/entry.ts

```diff
@@ -215,44 +144,43 @@ const main = async (args: Args, configArgs: Args): Promise<void> => {
     logger.info("  - Not serving HTTPS")
   }
 
-  if (httpServer.proxyDomains.size > 0) {
-    logger.info(`  - ${plural(httpServer.proxyDomains.size, "Proxying the following domain")}:`)
-    httpServer.proxyDomains.forEach((domain) => logger.info(`    - *.${domain}`))
-  }
-
-  if (serverAddress && !options.socket && args.open) {
-    // The web socket doesn't seem to work if browsing with 0.0.0.0.
-    const openAddress = serverAddress.replace(/:\/\/0.0.0.0/, "://localhost")
-    await open(openAddress).catch((error: Error) => {
-      logger.error("Failed to open", field("address", openAddress), field("error", error))
-    })
-    logger.info(`Opened ${openAddress}`)
+  if (args["proxy-domain"].length > 0) {
+    logger.info(`  - ${plural(args["proxy-domain"].length, "Proxying the following domain")}:`)
+    args["proxy-domain"].forEach((domain) => logger.info(`    - *.${domain}`))
   }
 
   if (args.link) {
     try {
-      await coderCloudBind(serverAddress!, args.link.value)
+      await coderCloudBind(serverAddress, args.link.value)
+      logger.info("  - Connected to cloud agent")
     } catch (err) {
       logger.error(err.message)
       ipcMain.exit(1)
     }
   }
+
+  if (serverAddress && !args.socket && args.open) {
+    // The web socket doesn't seem to work if browsing with 0.0.0.0.
+    const openAddress = serverAddress.replace(/:\/\/0.0.0.0/, "://localhost")
+    await open(openAddress).catch((error: Error) => {
+      logger.error("Failed to open", field("address", openAddress), field("error", error))
+    })
+    logger.info(`Opened ${openAddress}`)
```

Comment:
The `catch` method still allows the next logger line to print

```suggestion
    try {
      await open(openAddress)
    } catch (error) {
      logger.error("Failed to open", field("address", openAddress), field("error", error))
      ipcMain.exit(1)
    }

    logger.info(`Opened ${openAddress}`)
```